Sofa/framework/Geometry/test/Edge_test.cpp

```diff
@@ -230,6 +230,91 @@ TEST(GeometryEdge_test, pointBaryCoefs3f)
 }
 
 
+TEST(GeometryEdge_test, isPointOnEdge2f)
+{
+    const sofa::type::Vec2f a1{ 0.f, 0.f };
+    const sofa::type::Vec2f b1{ 2.f, 2.f };
+
+    // True case
+    const sofa::type::Vec2f p1{ 1.f, 1.f };
+    const auto res1 = sofa::geometry::Edge::isPointOnEdge(p1, a1, b1);
```

Comment:
To make sure that `isPointOnEdge` is `constexpr`, you can declare all the `res*` `constexpr`

Reply:
I tried and it seems the operator -  is not constexpr ?
```
1>C:\projects\sofa-src\Sofa\framework\Geometry\test\Edge_test.cpp(240,25): error C2131: expression did not evaluate to a constant
1>C:\projects\sofa-src\Sofa\framework\Type\src\sofa/type/Vec.h(412,31): message : failure was caused by a read of a variable outside its lifetime
1>C:\projects\sofa-src\Sofa\framework\Type\src\sofa/type/Vec.h(412,31): message : see usage of 'a1'
```

any idea?

Reply:
I tested and the operator- is really constexpr. I have no idea what could cause this error. I would drop the constexpr qualifier in this PR as I am not sure this is really useful.